### ISSUE TITLE
Stange define.xml error "ValueError: Missing required keyword argument _content in FormalExpression"

### ISSUE THREAD

**Issue by JozefAerts** [cdisc-org/cdisc-rules-engine#296]
With a define.xml (attached) we get the error "ValueError: Missing required keyword argument _content in FormalExpression", although the define.xml validates without problems against the XML-Schema.
The only thing I can think of is that the content of our FormalExpression contents is multiline, which is allowed.

Any idea? As the validation crashes to this issue, this is a "showstopper" for us in our application.

![Unbenannt](https://user-images.githubusercontent.com/25853332/224476730-56126676-1b4f-4763-bba2-2599d95b1e40.PNG)
[define.xml.txt](https://github.com/cdisc-org/cdisc-rules-engine/files/10948143/define.xml.txt)

**Comment by JozefAerts** [comment]
I haven't been able to reproduce it either today either with the code of the development branch downloaded at 2023-03-11, nor using the code of the main branch downloaded today (2023-03-21). 
I tried a few things like providing the false Define-xml version (-dv), but everything validated fine ...
Command used was: python core.py validate -s sdtmig -v 3-4 -d D:\MetaDataSubmissionGuide_2_0_SASXPTFiles_CORE_testing -dv 2.1 
OR python core.py validate -s sdtmig -v 3-4 -d D:\CDISC_Standards\CDISC_CORE\Tests\Define-XML_TranslatedText -dv 2.1
which has another define,xml which contains "FormalExpressions" within "MethodDef", and is equal to the define.xml that I uploaded here.
I will keep trying ...

**Comment by nhaydel** [comment]
@JozefAerts is it possible you ran it with a specific rule that uses the define xml?

**Comment by drewcdisc** [comment]
Need more details to include in another Sprint

**Comment by nhaydel** [comment]
@JozefAerts the issue is you have a FormalExpression with no content on line 20665, 20673, 20681 among other issues with the provided define xml. Please review your define xml file and open another issue if necessary

**Comment by JozefAerts** [comment]
Thanks @nhaydel !
An empty FormalExpression is allowed by the standard and XML-Schema. It is rather usual during development of mappings to SDTM that contain the mapping instructions in MethodDef.
I will generate a very small define.xml to test again, and create a new issue if necessary.
Many thanks again!

**Comment by nhaydel** [comment]
@JozefAerts Ahh ok. We are using @swhume odmlib library for parsing define xml. That library throws an error if FormalExpressions do not have content. Maybe that is something we can change in the library

**Comment by JozefAerts** [comment]
Yes, in the scope of a **final** regulatory submission, an empty FormalExpression doesn't make sense. However, during development of the mappings, it does.
